### DISCUSSION TITLE
namespace _mysql_affected_rows issue for Mac on Intel chip on py311

### DISCUSSION THREAD

**Discussion by TomMonkeyMan** [PyMySQL/mysqlclient#704] [Q&A]
hi team,

Recently I switched to py311 and encountered an import issue with:

```
Python 3.11.7 (main, Dec  4 2023, 18:10:11) [Clang 15.0.0 (clang-1500.1.0.2.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
import MySQLdb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/tiashi/Desktop/dataworks/shairflow/.toxenvs/tests-cn/lib/python3.11/site-packages/MySQLdb/__init__.py", line 17, in <module>
    from . import _mysql
ImportError: dlopen(/Users/tiashi/Desktop/dataworks/shairflow/.toxenvs/tests-cn/lib/python3.11/site-packages/MySQLdb/_mysql.cpython-311-darwin.so, 0x0002): symbol not found in flat namespace '_mysql_affected_rows'
```

I have checked this error and got a lot information on the platform version mismatch for Mac on M1/M2, like https://github.com/PyMySQL/mysqlclient/issues/496 . but mine is an old intel chip MacBook, not M*...

here's my env:
```
Python 3.11.7
(/Users/tiashi/Desktop/dataworks/shairflow/.toxenvs/tests-cn/bin/python: Mach-O 64-bit executable x86_64)
mysql  Ver 8.0.36 for macos13.6 on x86_64 (Homebrew)
(/usr/local/opt/mysql@8.0/bin/mysql: Mach-O 64-bit executable x86_64)
mysqlclient==2.2.0
```

I think my python311 has already run on i386, may I ask how can I fix it. any guidance would be helpful.

**Comment by TomMonkeyMan** [comment]
I installed the mysqlclient by adding local env with 
```
export MYSQLCLIENT_LDFLAGS=-L/usr/local/Cellar/mysql@8.0/8.0.36/lib
export MYSQLCLIENT_CFLAGS=-I/usr/local/Cellar/mysql@8.0/8.0.36/include/mysql
```

**Comment by methane** [comment]
https://github.com/PyMySQL/mysqlclient/issues/496#issuecomment-988474157

> After the reinstall of mysqlclient, I still got the error, it turns out pip will use a cached version when reinstall. I uninstall it and tried with pip install mysqlclient --no-cache-dir, and it worked for me.
